lib/printing/describe.go

```diff
@@ -29,6 +29,9 @@ func DescribeSink(dw printers.PrefixWriter, attribute, namespace string, sink *d
 	if sink == nil {
 		return
 	}
+	if len(attribute) == 0 {
```

Comment:
It looks like that this method is never called with an `attribute` that is different to "". So I just would remove that argument and maybe re-introduce it if there is ever a need for a different label than `Sink`.

Reply:
@rhuss : subscription CRUD uses it for DeadLetterSink etc

https://github.com/knative/client/blob/abb75e4da2c45534593bb2684dc3253eef3a7d91/pkg/kn/commands/subscription/describe.go#L105

Reply:
@daisy-ycguo : can you please update the function docstring to mention this behavior ? (like `'Sink' is used for attribute value if empty string is provided`)

Reply:
sure. I will update the docstring.

Reply:
I see. But maybe it's just better to require an explicit argument ? This could then also serve as documentation when calling the function (so just changing "" everywhere to "Sink") ? Not sure about this, so feel free to keep it .

Yet an alternative would be to move `attribute` to the end of the argument list and make it an optional parameter like `attribute ...string`. But I'm not dogmatic here, feel to chose what you think fits best.

Reply:
> But maybe it's just better to require an explicit argument ? This could then also serve as documentation when calling the function (so just changing "" everywhere to "Sink") ? Not sure about this, so feel free to keep it .

I was thinking same about it when I first saw this. As we are exporting this util, IMO we shouldn't carry any assumption and require explicit attribute value.

Reply:
I changed the code to make mandatory.